chapters/offenses.adoc

```diff
@@ -1,152 +1,169 @@
+* if the opposing team scores a <<Scoring Goals, valid goal>>.
+* at half-time and at overtime.
 
-If a robot other than the keeper touches the ball while being entirely inside its own defense area, the game is stopped and a <<Penalty Kick, penalty kick>> is awarded to the other team. The foul counter is not increased.
+The same no stop foul cannot be triggered again until the foul
+condition has stopped being violated or there has been 1 second since
+the foul was first triggered. This is to allow teams to adjust their
+robots' positions, ball speed or any other property that is causing
+the violation before being penalized additional times.
 
-==== Boundary Crossing
-A robot must not kick the ball over the field boundary such that the ball leaves the field.
+===== Attacker In Defense Area
+A robot must not touch the ball while being partially or fully inside the opponent <<Defense Area, defense area>>.
 
+NOTE: When the ball is <<Ball In And Out Of Play, out of play>>, the more strict rule <<Robot Too Close To Opponent Defense Area>> applies instead.
 
-=== Unsporting Behavior
-Unsporting behavior can lead to <<Yellow Card, yellow cards>>, <<Red Card, red cards>>, <<Penalty Kick, penalty kicks>>, a <<Forced Forfeit, forced forfeit>> or a <<Disqualification, disqualification>>. The human <<Referee, referee>> chooses an appropriate sanction, depending on the severity of the offense.
+===== Excessive Dribbling
+A robot must not <<Dribbling Device, dribble>> the ball further than 1 meter, measured linearly from the ball location where the dribbling started. A robot begins dribbling when it makes contact with the ball and stops dribbling when there is an observable separation between the ball and the robot.
 
-NOTE: If the referee is not sure which sanction to choose, he may confer with members of the <<Technical Committee, technical committee>> or the <<Organizing Committee, organizing committee>>.
+NOTE: Dribblers can still be used to dribble large distances with the ball as long as the robot periodically loses possession, such as kicking the ball ahead of it as human soccer players often do.
 
-Some examples of unsporting behavior are listed below.
+===== Ball Speed
+A robot must not accelerate the ball faster than 6.5 meters per second in 3D space.
 
-==== Damaging Other Robots
-It is not allowed to damage or modify robots of other teams.
+The <<Referee, human referee>> may decide to repeat the <<Kick-Off, kick-off>> or <<Free Kick, free kick>> on significant disturbances.
 
-==== Damaging The Field Or The Ball
-It is not allowed to damage or modify the field or the ball.
+NOTE: During <<Stop, stop>>, there is no automatic sanction for being too close to the ball. The referee may still punish a team for <<Unsporting Behavior,unsporting behavior>> by issuing a <<Yellow Card, yellow card>> if it does not respect the required distance. See <<Stop, stop>> for further explanation.
```

Comment:
This note seems to be really out of place here. It does not relate to the ball speed foul at all. If there is no accompanying foul, it might still be better to put it in the Fouls while ball out of play section

Reply:
good catch. This and the sentence before belongs to the Defender Too Close To Ball section. I've moved it.